h2o/src/main/java/org/apache/mahout/h2obindings/drm/H2OBCast.java

```diff
@@ -118,4 +118,16 @@ private T deserialize(byte buf[]) {
     }
     return ret;
   }
+
+  /**
+   * Stop broadcasting when called on driver side. Release any network resources.
+   *
+   */
+  @Override
+  public void close() throws IOException {
+
+    // TODO: review this. It looks like it is not really a broadcast mechanism but rather just a
```

Comment:
I am not sure how h20 does it but it looks like broadcast is just serialized to the backend with all requests, not really broadcasted. if so, there are no associated resources so method needs to do nothing. Anyhow it's something for Anand @avati  to look at.